### PR TITLE
Pythonised deleteKey test

### PR DESCRIPTION
**Pull request by wdednam**
Ported deleteKey TCL test to python.

- - -
This change is [Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/524)

### CODE REVIEW

**Review by wdednam — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/524)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @wdednam)

**Review by wdednam — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/524#-:-O9TOjg3CF0PSd90Nbt1:bnfp4nl)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @wdednam)

File: test/00/deleteKey.sh [hunk 1]
```diff
@@ -27,38 +27,83 @@ pass()
...
+    return None
+
+# Function to delete an item or wire at a given position
+def deleteKey(x, y):
+    # Check for a wire or item at the coordinates and delete it
+    if minsky.canvas.getWireAt(x, y):
+        minsky.canvas.deleteWire()
+        print(f"Wire deleted at ({x}, {y})")
+    elif minsky.canvas.getItemAt(x, y):
+        item = minsky.canvas.itemFocus()  # Try to focus on the item
```
Comment (highperformancecoder) [inline test/00/deleteKey.sh:72]: This part is a bit strange - getItemAt populates canvas.item, not canvas.itemFocus. But this doesn't invalidate the test.
`bool getItemAt(float x, float y) override {return (item=itemAt(x,y)).get();}`

File: test/00/deleteKey.sh [hunk 2]
```diff
@@ -27,38 +27,83 @@ pass()
...
+                if minsky.canvas.getItemAt(x, y):
+                    return (x, y)
+                else:
+                    print(f"Failed to focus item at ({x}, {y}). Trying manual fallback.")
+                    return (x, y)  # Return coordinates anyway for manual handling
+        except Exception as e:
+            last_error = e
+            pass
+    if last_error:
+        print(f"Error while finding variable '{name}' at last known coordinates: {last_error}")
```
Comment (highperformancecoder) [inline test/00/deleteKey.sh:62]: You could put this code in the "except" block

**Review by highperformancecoder — APPROVED** [review]

**Pull request merged**